### PR TITLE
Fl20 users orders

### PR DESCRIPTION
**Pull request by Danatlol**
блин, комит с другого логина пошел(

### CODE REVIEW

**Comment by Danatlol** [comment]
[FL20_users_orders](https://trello.com/c/HD67yR2q/57-fl20usersorders)

File: src/app/components/users-orders/order-details/order-details.component.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (Evittta) [inline src/app/components/users-orders/order-details/order-details.component.css]: don't leave an empty line at the start of file

File: src/app/components/users-orders/order-details/order-details.component.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<mat-expansion-panel>
+
+  <mat-expansion-panel-header>
+    <mat-panel-title>
+      Status:&nbsp;<b>{{order.status}}</b>
+    </mat-panel-title>
+    <mat-panel-description>
+      <b fxHide.lt-md="true">{{order.restaurant}}&nbsp;on&nbsp;</b><b>{{order.date}}&nbsp;{{order.time}}</b><b fxHide.lt-md="true">&nbsp;on&nbsp;{{order.persons}}&nbsp;persons</b>.
```
Comment (Evittta) [inline src/app/components/users-orders/order-details/order-details.component.html]: avoid long code line

File: src/app/components/users-orders/order-details/order-details.component.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  <hr>
+  <div fxLayout="row" fxLayoutAlign="flex-start center" fxLayoutGap="30px" fxLayout.lt-md="column">
+
+    <div fxLayout="column" fxLayoutAlign="center center" fxLayoutGap="8px">
+      <mat-icon class="big-font" color="primary">room_service</mat-icon>
+      <button mat-raised-button color="primary" (click)="onOrderHandler()">Order again</button>
+    </div>
+    <div fxFlexAlign="stretch" fxLayout="column" fxLayoutAlign="space-between stretch">
+      <p fxHide="true" fxHide.lt-md="false" class="center-block">
+          <b>{{order.restaurant}}</b>&nbsp;on&nbsp;<b>{{order.date}}&nbsp;{{order.time}}</b>&nbsp;on&nbsp;<b>{{order.persons}}</b>&nbsp;persons.
```
Comment (Evittta) [inline src/app/components/users-orders/order-details/order-details.component.html]: avoid long code line

File: src/app/components/users-orders/past-order-card/past-order-card.component.css [hunk 4]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (Evittta) [inline src/app/components/users-orders/past-order-card/past-order-card.component.css]: empty line at start of the file

File: src/app/components/users-orders/past-order-card/past-order-card.component.html [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+        <mat-expansion-panel-header>
+          <mat-panel-title>
+            Status: {{order.status}}
+          </mat-panel-title>
+          <mat-panel-description>
+            Click to see the answer.
+          </mat-panel-description>
+        </mat-expansion-panel-header>
+        <p>{{order.answer}}</p>
+      </mat-expansion-panel> -->
```
Comment (Evittta) [inline src/app/components/users-orders/past-order-card/past-order-card.component.html]: comment

File: src/app/components/users-orders/users-orders.component.css [hunk 6]
```diff
@@ -0,0 +1,31 @@
+
+
```
Comment (Evittta) [inline src/app/components/users-orders/users-orders.component.css]: two empty line

File: src/app/main/coord-picker/coord-picker.component.ts [hunk 7]
```diff
@@ -45,6 +45,7 @@ export class CoordPickerComponent implements OnInit {
 
   public onMapClick($event) {
     const coord = new Coordinates($event.coords.lat, $event.coords.lng);
+    // this.infoWindow._closeInfoWindow();
```
Comment (Evittta) [inline src/app/main/coord-picker/coord-picker.component.ts]: comment

File: src/app/store/search-places/search-places.effects.ts [hunk 8]
```diff
@@ -43,47 +51,118 @@ export class SearchPlacesEffects {
...
+//     let request = {
+//       location: coords,
+//       radius: searchPlacesStore.searchRadius,
+//       type: searchPlacesStore.searchPlaceType,
+//       openNow: searchPlacesStore.searchIsOpenNow
+//     };
+//     return this.searchPlacesService.getSearchPlacesObservable()(request);
+//   })
+//   .map(response => new SearchPlaces.StopRadarSearch(response))
+//   .catch(() => of(new SearchPlaces.StopRadarSearch([])));
```
Comment (Evittta) [inline src/app/store/search-places/search-places.effects.ts]: comment

File: src/app/store/search-places/search-places.effects.ts [hunk 9]
```diff
@@ -43,47 +51,118 @@ export class SearchPlacesEffects {
...
+//   )
+//   .switchMap(action => {
+//     return this.searchPlacesService.getCurrentLocationCoordinates()();
+//   })
+//   .map(coord => new SearchPlaces.StopSearchMapStartCoordinates(coord))
+//   .catch(() =>
+//     of(
+//       new SearchPlaces.StopSearchMapStartCoordinates(new Coordinates(49, 24))
+//     )
+//   );
```
Comment (Evittta) [inline src/app/store/search-places/search-places.effects.ts]: comment

File: src/app/store/search-places/search-places.effects.ts [hunk 10]
```diff
@@ -43,47 +51,118 @@ export class SearchPlacesEffects {
...
+//   )
+//   .switchMap(action => {
+//     return this.searchPlacesService.getPlaceByIdObservable()({
+//       placeId: action.payload.searchPlaceId
+//     });
+//   })
+//   .map((placeDetails: PlaceDetails) => {
+//     return new SearchPlaces.StopPlaceDetailsSearch(placeDetails);
+//   });
+// }
```
Comment (Evittta) [inline src/app/store/search-places/search-places.effects.ts]: comment

**Pull request merged**